### DISCUSSION TITLE
Migration guide to stable 3.0.0

### DISCUSSION THREAD

**Discussion by sikelio** [lepoco/wpfui#941] [Q&A] [answered]
Hi,

I've created a project with version 3.0.0-preview-4 and now that the the 3.0.0 it released i wanted to know if there is any migration guide, or if i have to start from scratch a new project with the stable release.

**Comment by StoiaCode** [comment]
Same with 2.1.0 to 3.0.0 btw.

**Comment by pomianowski** [comment, marked as answer]
Hey @sikelio, there are no such guides for now, the 3.0.+ release is still under development and the only official code pattern is the Wpf.Ui.Gallery application
https://github.com/lepoco/wpfui/tree/main/src/Wpf.Ui.Gallery

**Reply by sikelio** [reply]
Hey @pomianowski thank your answer
